media_kit_video/linux/video_output_manager.cc

```diff
@@ -13,18 +13,21 @@ struct _VideoOutputManager {
   GHashTable* video_outputs;
   FlTextureRegistrar* texture_registrar;
   FlView* view;
+  GLRenderThread* gl_render_thread;
 };
 
 G_DEFINE_TYPE(VideoOutputManager, video_output_manager, G_TYPE_OBJECT)
 
 static void video_output_manager_init(VideoOutputManager* self) {
   self->video_outputs = g_hash_table_new_full(g_direct_hash, g_direct_equal,
                                               nullptr, g_object_unref);
+  self->gl_render_thread = new GLRenderThread();  // Dedicated GL render thread
```

Comment:
Lets use `g_autoptr` (similar to VideoOutputManager itself). It's a convention to use std:: smart pointers on Windows & GLib on Windows in the project.